front-end/src/UI/Atoms/LogInButton/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from 'react-bootstrap/Button';
+
+const BlackBox = styled.div`
+  background-color: black;
+  width: 300px;
+  height: 80px;
+`;
+
+function LogInButton() {
+  return (
+    <BlackBox>
+      <Button variant="light" size="lg">
+        LOGIN
+      </Button>{' '}
```

Comment:
여기에 `{' '}` 가 있는 이유가 있나요? 삭제해도 될 것 같아요.

```suggestion
      </Button>
```

이렇게..